projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts

```diff
@@ -187,7 +187,7 @@ function addRootTuiProvidersToBootstrapFn(
         pushToObjectArrayProperty(
             bootstrapOptions,
             `providers`,
-            `importProvidersFrom(TuiRootModule, BrowserAnimationsModule)`,
```

Comment:
I think we need to add `provideAnimations()` if it's missing.

Reply:
fixed